packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,479 @@ Before installing Chainweb, you should verify that your computer meets the follo
+You might experience some unexpected behavior when running a node on ARM64 architecture.
+If you want to run a Chainweb node on Microsoft Windows, you should note that this isn't a supported configuration and requires you to install the latest Windows Services for Linux (WSL) software.
+If you choose to proceed after installing WSL, you can install the `chainweb-node` software in the WSL virtual environment rather than using the native Windows operating system. 
+
+### Incoming and outgoing network rules
+
+In addition to a publicly-accessible IP address, nodes must be able to communicate with peers and accept incoming messages from other computers in the network.
+If your computer is behind a firewall or is a virtual guest connecting to the internet through a host computer, you should review your network configuration and open required ports to allow inbound and outbound traffic. 
+For example, you might need to configure port forwarding for port 1789 on your router or host computer to allow inbound connections from remote nodes.
+
 ## Installation options
 
 There are several options for setting up a Chainweb node in a physical or virtual environment.
 For example, you can run a Chainweb node image in a Docker container or build the binaries directly from the source code.
+You can install release binaries directly on a physical Linux server or run them using the infrastructure from a cloud services provider.
+
+For more information about your installation options, see the following topics:
+
+- [Install release binaries](#install-release-binaries)
+- [Run in a Docker container](#run-in-a-docker-container)
+- [Build from source code](#build-from-source)
+
+## Install release binaries
+
+You can download compressed archive files with `chainweb-node` release binaries for Ubuntu Linux directly from the [Releases](https://github.com/kadena-io/chainweb-node/releases) pages in the [chainweb-node](https://github.com/kadena-io/chainweb-node/) repository.
+If you have Ubuntu 20.0.4 or Ubuntu 22.04 on a physical or virtual machine, downloading the binary is the most straightforward installation path.
+
+To install from a release archive:
+
+1. Open a terminal shell on the physical or virtual host with the Ubuntu Linux operating system.
+
+2. Update the system with the latest software by running the following command:
+   
+   ```bash
+   sudo apt update && sudo apt upgrade
+   ```
+
+3. Install the required packages by running the following command:
+   
+   ```bash
+   sudo apt-get install ca-certificates libgmp10 libssl3 libsnappy1v5 zlib1g liblz4-1 libbz2-1.0 libgflags2.2 zstd
+   ```
+
+   Note that Ubuntu 20.04 requires the `libssl1.1` package instead of the `libssl3` package. 
+   This package has been deprecated in Ubuntu 22.04.
+   If your computer uses the Ubuntu 20.04 operating system, install the `libssl1.1` package instead of the `libssl3` package. 
```

Comment:
Maybe the last line here is all that's needed? Or maybe two separate command lines, one for each Ubuntu version, would be clearer.